com.hoptodesk.HopToDesk.yml

```diff
@@ -0,0 +1,52 @@
+id: com.hoptodesk.HopToDesk
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: hoptodesk
+icon: share/icons/hicolor/scalable/apps/hoptodesk.svg
+appdata:
+  license: FSFAP
+  url: https://hoptodesk.com
+  metadata_license: FSFAP
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - xdotool.json
+  - name: HopToDesk
+    buildsystem: simple
+    build-commands:
+      - bsdtar -zxvf hoptodeskdeb.deb
+      - tar -xvf ./data.tar.xz
+      - cp -r ./usr/*  /app/
+      - ls /app/lib
+      - cp /app/lib/libsciter-gtk.so  /app/bin/
+      - chmod 755 /app/bin/libsciter-gtk.so
+      - mkdir -p /app/bin
+      - mv /app/share/applications/hoptodesk.desktop /app/share/applications/com.hoptodesk.HopToDesk.desktop
+      - sed -i '/^Icon=/ c\Icon=com.hoptodesk.HopToDesk' /app/share/applications/com.hoptodesk.HopToDesk.desktop
+      - mv /app/share/icons/hicolor/128x128/128x128.png /app/share/icons/hicolor/128x128/com.hoptodesk.HopToDesk.png
+      - "for size in 16 24 32 48 64 128 256 512; do\n rsvg-convert -w $size -h $size\
+        \ -f png -o $size.png logo.svg\n install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/com.hoptodesk.HopToDesk.png\n\
+        \ done"
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share/gtk-doc
+    sources:
+      - type: file
+        path: hoptodeskdeb.deb
+
+      - type: file
+        dest-filename: logo.svg
+        url: https://gitlab.com/hoptodesk/hoptodesk/-/raw/ab5356ba2ce36e3ee0bab6e30cc051ad07c49ecd/res/logo.svg
+        sha256: 7107616c5649a5bd952479f02fb4619f764578fa8bcf54c4ea4bc5466244895b
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
You don't need all 3. If the app supports wayland use fallback-x11+wayland if not only x11.